indexer-app/resources/schemas/dataqualitysummary/v1.0.0/index.json

```diff
@@ -0,0 +1,17 @@
+{
+    "MetadataSpecification": {
+        "URL": "https://cdn.earthdata.nasa.gov/generics/dataqualitysummary/v1.0.0",
+        "Name": "DataQuailtySummary",
+        "Version": "1.0.0"
+    },
+    "SubConceptType": "DQS",
```

Comment:
we need to add this to the schema.json file

Reply:
What do we need to add to the schema.json file?  This shouldn't go into the actual data quality summary schema.